src/js/background/index.html

```diff
@@ -14,9 +14,11 @@
     ]
 -->
     <script type="text/javascript" src="backgroundLogic.js"></script>
+    <script type="text/javascript" src="sync.js"></script>
     <script type="text/javascript" src="assignManager.js"></script>
     <script type="text/javascript" src="badge.js"></script>
     <script type="text/javascript" src="identityState.js"></script>
     <script type="text/javascript" src="messageHandler.js"></script>
+    <script type="text/javascript" src="test.js"></script>
```

Comment:
Ideally test files shouldn't ever be included in the extension output.

Reply:
Working on that after/during QA

Reply:
Opened https://github.com/kendallcorner/multi-account-containers/pull/1 to help with migrating the tests into the suite.

Reply:
Thanks!